include/openssl/sslerr.h

```diff
@@ -303,6 +303,7 @@ int ERR_load_SSL_strings(void);
 # define SSL_F_TLS_CONSTRUCT_STOC_SESSION_TICKET          460
 # define SSL_F_TLS_CONSTRUCT_STOC_STATUS_REQUEST          461
 # define SSL_F_TLS_CONSTRUCT_STOC_SUPPORTED_GROUPS        544
+# define SSL_F_TLS_CONSTRUCT_STOC_SUPPORTED_VERSIONS      548
```

Comment:
```
$ if $make update; then echo -e '+\057 MAKE UPDATE OK'; else echo -e '+\057 MAKE UPDATE FAILED'; false; fi; git diff --exit-code
SSL function code 548 collision at SSL_F_TLS_CONSTRUCT_STOC_SUPPORTED_VERSIONS
make: *** [errors] Error 25
+/ MAKE UPDATE FAILED
The command "if $make update; then echo -e '+/ MAKE UPDATE OK'; else echo -e '+/ MAKE UPDATE FAILED'; false; fi; git diff --exit-code" exited with 0.
```

Reply:
Strange...I don't see this collision?

Reply:
548 used by SSL_F_TLS_CONSTRUCT_STOC_MAXFRAGMENTLEN

Reply:
I just copy-paste a piece of a Travis failed jobs.
As I say in #1008, the reworked `util/mkerr.pl` script don't break (all) the build(s) after a `make update` when there is some duplicated IDs .

Reply:
Please look for "make update" in this log, for example:
https://travis-ci.org/openssl/openssl/jobs/299726329

Reply:
That is very interesting and highlights something about the way that Travis works that I had not appreciated. The value `SSL_F_TLS_CONSTRUCT_STOC_MAXFRAGMENTLEN` does not exist and there is no conflict *in this branch*. However it *does* exist in the master branch. It appears to me that travis is doing a merge of this branch with master before it attempts to build. I had always assumed that the build took place on this branch only.

Reply:
Yes, github provides a candidate-merge commit for a pull request, which is why travis refuses to provide any results when there is a merge conflict -- the candidate-merge commit does not exist in a usable state.